packages/devtools-evm-hardhat/package.json

```diff
@@ -66,6 +66,7 @@
     "@swc/core": "^1.4.0",
     "@swc/jest": "^0.2.36",
     "@types/jest": "^29.5.12",
+    "ethers": "^5.7.2",
```

Comment:
Instead of `ethers` please use `@ethersproject/providers`, it's already in the dependencies

Reply:
Ah great catch -- updated!